recipes/numpy/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set version="1.11.0" %}
+
+package:
+  name: numpy
+  version: {{ version }}
+
+source:
+  fn: numpy-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/n/numpy/numpy-{{ version }}.tar.gz
+  md5: bc56fb9fc2895aa4961802ffbdb31d0b
+  # patches:
+    # - osx-dist.patch          # [osx]
+
+build:
+  number: 0
+  features:
+    - blas_openblas
+
+requirements:
+  build:
+    - cython
+    - gcc
+    - openblas
+    - python
+    - setuptools
```

Comment:
Just as FYI for anyone that finds this curious. NumPy started using `setuptools` in 1.11.0. There were just too many issues installing this with `setup.py` so this became the simpler option.